src/components/requests/features.info.vue

```diff
@@ -1,19 +1,31 @@
 <template>
   <div ref="GetFeatureInfoRequest">
+    To show feature info box for a certain layer you must ensure first that the layer visibility is set to true and then proceed to send GetFeatureInfoRequest.
+    <br><br>
+    1. Set visibility of wanted layer as true. In this example visibility is set to true already.
+    <CodeSnippet snippet="channel.postRequest('MapModulePlugin.MapLayerVisibilityRequest', [492, true]);
```

Comment:
You can have the code inside the tag body :) It makes no sense in doing it in the tag attribute.